docs/Splashkit/Modules/Machine Learning/SRS.md

```diff
@@ -0,0 +1,113 @@
+# Software Requirements Specifications Document
+
+### Machine Learning for Splashkit Modules
+
+# Product Purpose
```

Comment:
Ideally there is only 1 H1 per file.
I would make this a H2 and the same for any below

Reply:
Yep ok that should be fixed now